Duplicati/Library/Utility/JoinedFilterExpression.cs

```diff
@@ -73,9 +75,9 @@ public static IFilter Join(IFilter first, IFilter second)
         public override string ToString()
         {
             if (this.First.Empty)
-                return this.Second.ToString();
+                return this.Second.ToString() ?? throw new InvalidOperationException("Second is null");

```

Comment:
This looks off. Can `.ToString()` ever be null? The `First` and `Second` properties are never null.